internal/errors/grpc.go

```diff
@@ -19,33 +19,39 @@ package errors
 
 var (
 
-	// gRPC
-
+	// ErrgRPCClientConnectionClose represents a function to generate an error that the gRPC connection couldn't close.
 	ErrgRPCClientConnectionClose = func(name string, err error) error {
```

Comment:
I think it should be `ErrgRPCClientConnectionClose`

Reply:
I could not understand your opinion ...
I'm sorry I don't understand.:bow;
so could you please tell me about this more clearly?? :bow:


Does this mean I need to modify the function name? 🤔

Reply:
sorry it was my mistake. I have modified the comment above.

Reply:
thank you. 🙏 
But I can not see the modified your comment above...:thinking:
It may not have been modified? 🤔 
so could you please resend it ?? :bow:

Reply:
Sorry I should describe here.
Here is the suggestion.
```suggestion
	ErrGRPCClientConnectionClose = func(name string, err error) error {
```

Reply:
thank you. I get it 👍 
I will fix it soon 🙏

Reply:
Does this change affect current code?
do you already check?

Reply:
@vankichi 
thank you for your comment.
I think this change does not affect to current code.  🤔 
I searched for a usage points before, but I couldn't find it.
The following is the searched result :bow:
https://github.com/vdaas/vald/search?q=ErrgRPCClientConnectionClose

Reply:
@vankichi 
thank you for your comment.
I think this change does not affect to current code.  🤔 
> do you already check?

Yes, I did.
I searched for a usage points before, but I couldn't find it.
The following is the searched result :bow:
https://github.com/vdaas/vald/search?q=ErrgRPCClientConnectionClose